angular/projects/admin/src/app/admin/navigation/profile-widget/profile-widget.component.ts

```diff
@@ -22,4 +23,11 @@ export class ProfileWidgetComponent implements OnInit {
     this.appAuthService.logOut();
   }
 
+  private async _reloaduser() {
+    const payload = await this.appAuthService.reloadUserService();
```

Comment:
Rename `reloadUserService` -> `reloadUser`

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/195/commits/87684dd3dcaa2d579fc888e3fc60e9e159a3dc2e